app/gcl/hands-on/gke-private-cluster-security-guide/page.css

```
@@ -0,0 +1,448 @@
+.gke-security-guide-page .mermaid-wrap {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    background: var(--color-card, #0d1b2e);
+    border: 1px solid #1c2d42;
+    border-radius: var(--radius-lg, 12px);
+    padding: 28px 16px;
+    overflow-x: auto;
+}
+
+.gke-security-guide-page .mermaid-wrap :global(svg) {
```

Stylelint (17.14.1)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)